.github/workflows/snyk.yml

```diff
@@ -0,0 +1,30 @@
+name: Snyk vulenrability scan
+
+on: [push]
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '6.x'
+      - name: Restore dependencies
+        run: dotnet restore
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/dotnet@master
+        continue-on-error: true # To make sure that SARIF upload gets called
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          args: 
+            --all-projects 
+            --sarif-file-output=snyk.sarif 
+            --severity-threshold=high
+      - name: List files for debugging
```

Comment:
Is this debugging important in the `snyk` pipeline?

Reply:
Leftover from debugging. Removed.